nevergrad/benchmark/experiments.py

```diff
@@ -570,6 +618,24 @@ def yabbob(seed: tp.Optional[int] = None, parallel: bool = False, big: bool = Fa
         for num_blocks in [1]
         for d in ([100, 1000, 3000] if hd else [2, 10, 50])
     ]
+    assert constraints < 8, "We have only four possible constraints."
+    for func in functions:
+        if constraints > 0 and constraints <= 4:
+            func.parametrization.register_cheap_constraint(_positive_sum_float)
+        if constraints > 1 and constraints <= 5:
+            func.parametrization.register_cheap_constraint(_positive_diff_float)
+        if constraints > 2 and constraints <= 6:
+            func.parametrization.register_cheap_constraint(_positive_second_diff_float)
+        if constraints > 3 and constraints <= 7:
+            func.parametrization.register_cheap_constraint(_ball_float)
+        if constraints > 4:
+            func.parametrization.register_cheap_constraint(_positive_sum)
+        if constraints > 5:
+            func.parametrization.register_cheap_constraint(_positive_diff)
+        if constraints > 6:
+            func.parametrization.register_cheap_constraint(_positive_second_diff)
+        if constraints > 7:
+            func.parametrization.register_cheap_constraint(_ball)
```

Comment:
please restructurate using some kind of for loop, this is too harad to read.
using something like constraints = [_positive_sum_float, ...]
and then selecting the ones to register should help.

Reply:
Done.